### ISSUE TITLE
DeprecationWarning: datetime.datetime.utcnow()

### ISSUE THREAD

**Issue by Siecje** [uniphil/feedwerk#5]
```
venv/lib/python3.12/site-packages/feedwerk/atom.py:166: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
    self.updated = dates[-1] if dates else datetime.utcnow()
```

I know this project supports Python2 so I'm wondering what is the path forward.

**Comment by Siecje** [comment]
Thanks for working on this!

**Comment by uniphil** [comment]
you're very welcome, and thank you for reporting it! i was just trying to remember what to do to update the release on pypi, and then [noticed it already happened](https://pypi.org/project/feedwerk/)?! past me setting up automation, wild.

let me know if you have any issues with it :)

edit: fixed via #6
